tests/components/mobile_app/test_webhook.py

```diff
@@ -160,8 +160,10 @@ def store_event(event):
     assert resp.status == 200
 
     json = await resp.json()
-    assert len(json) == 1
-    assert json[0]["entity_id"] == "zone.home"
+    assert len(json) == 2
+    zones = sorted(json, key=lambda entry: entry["entity_id"])
+    assert zones[0]["entity_id"] == "zone.home"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5195031)